src/details/ArborX_DetailsTreeTraversal.hpp

```diff
@@ -275,17 +275,14 @@ struct TreeTraversal<BVH, Predicates, Callback, NearestPredicateTag>
                                                       buffer.size()));
 
     using PairNodePtrDistance = Kokkos::pair<Node const *, float>;
-    Stack<PairNodePtrDistance> stack;
-    // Do not bother computing the distance to the root node since it is
-    // immediately popped out of the stack and processed.
-    stack.emplace(bvh_.getRoot(), 0.);
+    PairNodePtrDistance stack[64];
+    auto *stack_ptr = stack;
+    *stack_ptr++ = {nullptr, 0.f};
```

Comment:
```suggestion
    *stack_ptr++ = {NULL, 0.f};
```
for consistency

Reply:
Changed all `NULL` to `nullptr` instead.